assets/templates/kubernetes.pod/gotext_multiline.tpl

```diff
@@ -198,332 +199,8 @@
-         "10.232.83.1",
-         "fe80::b4c3:ebff:febe:a01",
-         "10.232.83.1",
-         "fe80::858:8ff:fee1:a643",
-         "10.232.83.1",
-         "fe80::f4af:c9ff:feec:1a8a",
-         "10.232.83.1",
-         "fe80::94fa:6dff:feed:b3ab",
-         "10.232.83.1",
-         "fe80::42c:c2ff:fe0e:ff4b",
-         "10.232.83.1",
-         "fe80::684b:4eff:fec9:face",
-         "10.232.83.1",
-         "fe80::cc97:6cff:fe58:40e2",
-         "10.232.83.1",
-         "fe80::d806:9aff:fe33:79d7",
-         "10.232.83.1",
-         "fe80::a89c:f4ff:fef0:764",
-         "10.232.83.1",
-         "fe80::3032:c7ff:fe8c:5411",
-         "10.232.83.1",
-         "fe80::a1:c8ff:fe0b:5edf",
-         "10.232.83.1",
-         "fe80::1073:5fff:fe4e:81f7",
-         "10.232.83.1",
-         "fe80::54d2:97ff:feaa:9a02",
-         "10.232.83.1",
-         "fe80::d0db:58ff:fe48:af07",
-         "10.232.83.1",
-         "fe80::2891:aaff:fed5:a33a",
-         "10.232.83.1",
-         "fe80::44f1:c8ff:fee1:3f37",
-         "10.232.83.1",
-         "fe80::d4c7:6aff:feab:2495",
-         "10.232.83.1",
-         "fe80::d8da:dcff:fef5:8384",
-         "10.232.83.1",
-         "fe80::c494:b2ff:fed7:b34e",
-         "10.232.83.1",
-         "fe80::806a:73ff:fed9:eac",
-         "10.232.83.1",
-         "fe80::8879:72ff:fe4a:604a",
-         "10.232.83.1",
-         "fe80::a43a:53ff:fe19:d4fa",
-         "10.232.83.1",
-         "fe80::78a0:31ff:fe66:6aa4",
-         "10.232.83.1",
-         "fe80::a3:48ff:feec:2141",
-         "10.232.83.1",
-         "fe80::ecec:71ff:fef6:7f16",
-         "10.232.83.1",
-         "fe80::5c58:24ff:fe8f:101a",
-         "10.232.83.1",
-         "fe80::606f:dcff:fe9b:b9a5",
-         "10.232.83.1",
-         "fe80::ac11:3cff:fed5:d25",
-         "10.232.83.1",
-         "fe80::189f:8eff:fedf:a011",
-         "10.232.83.1",
-         "fe80::f406:54ff:fe94:b3a3",
-         "10.232.83.1",
-         "fe80::bc07:e2ff:fea4:ae01",
-         "10.232.83.1",
-         "fe80::1433:3bff:fe53:5b66",
-         "10.232.83.1",
-         "fe80::a89b:77ff:feae:bcc6",
-         "10.232.83.1",
-         "fe80::24a3:40ff:fe0d:6ba0",
-         "10.232.83.1",
-         "fe80::e050:4eff:fe47:58e2",
-         "10.232.83.1",
-         "fe80::28d6:97ff:fe96:5615",
-         "10.232.83.1",
-         "fe80::88b:57ff:fee3:bc1e",
-         "10.232.83.1",
-         "fe80::184d:f8ff:fe96:4109",
-         "10.232.83.1",
-         "fe80::c898:ebff:fe50:a559",
-         "10.232.83.1",
-         "fe80::8400:edff:fea3:e1ad",
-         "10.232.83.1",
-         "fe80::34f7:7aff:fe03:de89",
-         "10.232.83.1",
-         "fe80::829:49ff:fee5:3ea5",
-         "10.232.83.1",
-         "fe80::9cc1:ccff:fe21:8cd2",
-         "10.232.83.1",
-         "fe80::94cf:87ff:fe1f:7a92",
-         "10.232.83.1",
-         "fe80::5d:14ff:fec1:2d3a"
-      ],
-      "name": "{{ $agentName }}",
+      "name": "{{- $nodeid := generate "nodeid" -}}",
```

Comment:
beware, if you invoke `generate "nodeid"` two times in the same template you should end up with two different values

you should: because looking at the screenshot you posted it does not seems so, but this is something to investigate, since it seems to be bug :)

Reply:
ok once per file. Then I refer to nodeid variable